mappings/net/minecraft/entity/ai/AiGoalJumpOutOfLiquid.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS vu net/minecraft/entity/ai/AiGoalJumpOutOfLiquid
```

Comment:
Would `AiGoalSwim` be better? Looks like you also mapped a method `canSwim` which is probably related to this goal.